experimental/python/codegen/codegen/jsonschema_patch.py

```diff
@@ -3,43 +3,15 @@
 from codegen.jsonschema import Schema
 
 REMOVED_FIELDS = {
-    "jobs.RunJobTask": {
-        # all params except job_parameters should be deprecated and should not be supported
-        "jar_params",
-        "notebook_params",
-        "python_params",
-        "spark_submit_params",
-        "python_named_params",
-        "sql_params",
-        "dbt_commands",
-        # except pipeline_params, that is not deprecated
-    },
-    "jobs.TriggerSettings": {
-        # Old table trigger settings name. Deprecated in favor of `table_update`
-        "table",
+    # TODO remove as a follow-up
+    "jobs.Task": {
+        "dashboard_task",
+        "power_bi_task",
```

Comment:
Why remove?

Reply:
Separately is fine, keeps this one focused.

The comment didn't suggest diff minimization, hence the question.